src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search--form search--form__big"
+      action="#"
+      method="post"
+      data-qa="big">
+      <input
+        class="search search--big"
+        type="text"
+        placeholder="Try &#34Los Angeles&#34"
+        data-qa="keypress">
```

Comment:
Closed bracked should be on the same line as open - fix all cases
```suggestion
    <form
      class="search--form search--form__big"
      action="#"
      method="post"
      data-qa="big"
    >
      <input
        class="search search--big"
        type="text"
        placeholder="Try &#34Los Angeles&#34"
        data-qa="keypress"
      >
```